### PR TITLE
"Manage bookmarks" sample

### PR DESCRIPTION
**Pull request by PaulAllanSturm**
Newly implemented "Manage bookmarks" sample.

### CODE REVIEW

File: lib/samples/manage_bookmarks/manage_bookmarks_sample.dart [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+        20.0,
+        0.0,
+        20.0,
+        max(
+          20.0,
+          View.of(context).viewPadding.bottom /
+              View.of(context).devicePixelRatio,
+        ),
+      ),
+      child: SingleChildScrollView(
```
Comment (jenmerritt) [inline lib/samples/manage_bookmarks/manage_bookmarks_sample.dart:105]: The only thing about the scroll being here is that the title and 'x' also scroll. It feels like the list should scroll, but you should always be able to close the sheet if that makes sense?

File: lib/samples/manage_bookmarks/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Manage bookmarks
+
+Access and create bookmarks on a map.
+
+![Image of manage bookmarks](manage_bookmarks.png)
+
+## Use case
+
+Bookmarks are used for easily storing and accessing saved locations on the map. Bookmarks are of interest in educational apps (e.g., touring historical sites) or more specifically, for a land management company wishing to visually monitor flood levels over time at a particular location. These locations can be saved as bookmarks and revisited easily each time their basemap data has been updated (e.g. working with up-to-date satellite imagery to monitor water levels).
```
Comment (jenmerritt) [inline lib/samples/manage_bookmarks/README.md:9]:
```suggestion
Bookmarks are used for easily storing and accessing saved locations on the map. Bookmarks are of interest in educational apps (e.g. touring historical sites) or more specifically, for a land management company wishing to visually monitor flood levels over time at a particular location. These locations can be saved as bookmarks and revisited easily each time their basemap data has been updated (e.g. working with up-to-date satellite imagery to monitor water levels).
```

File: lib/samples/manage_bookmarks/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## How to use the sample
+
+The map in the sample comes pre-populated with a set of bookmarks. To access a bookmark and move to that location, tap on a bookmark's name from the 'Bookmarks' list. To add a bookmark, pan and/or zoom to a new location and tap the 'Add Bookmark' button. Enter a unique name for the bookmark and tap Add, and the bookmark will be added to the list.
```
Comment (jenmerritt) [inline lib/samples/manage_bookmarks/README.md:13]:
```suggestion
The map in the sample comes pre-populated with a set of bookmarks. To access a bookmark and move to that location, tap on a bookmark's name from the 'Bookmarks' list. To add a bookmark, pan and/or zoom to a new location and tap the 'Add Bookmark' button. Enter a unique name for the bookmark and tap 'Add', and the bookmark will be added to the list.
```

**Review by kmueller-gis — APPROVED** [review]

File: lib/samples/manage_bookmarks/manage_bookmarks_sample.dart (hunk 1 above)
Comment (PaulAllanSturm) [reply lib/samples/manage_bookmarks/manage_bookmarks_sample.dart]: I adjusted the widgets a bit so now the bookmarks scroll by themselves

**Review by jenmerritt — APPROVED** [review]
Way better UI 👍 so maxHeight was the key, nice.

**Pull request merged**